Scripts/Python/clftNpcZandi.py

```diff
@@ -86,9 +86,9 @@
 IgnoreTime = 120
 PageTurnInterval = 120
 
-TimerID = xEnum.Enum("IgnoreFinished = 10, TurnPage")
+TimerID = enum.IntEnum("TimerID", {"IgnoreFinished": 10, "TurnPage": 11})
```

Comment:
Maybe `start=10` instead?

Reply:
I considered it but I thought it could be too easily read over as "start=0" since that occurs in some places, though indeed it would be more consistent...